### DISCUSSION TITLE
Charge power limited at low SOC

### DISCUSSION THREAD

**Discussion by Dr-Gigavolt** [Louisvdw/dbus-serialbattery#344] [General]
Hello, I finished my installation couple of days ago. It consists of:

- Fronius Symo 15kW from existing system
- SmartSolar MPPT 250/100
- 3x Victron Multi-II 48/5000
- 2x LTO Battery 22S x 5P + Yinlong 66160H 2.3V 40Ah + JK BMS
- Raspberry Pi 3B with Venus 2.92
- Serial Battery driver 0.14

![console](https://user-images.githubusercontent.com/115069923/206924797-9f8f4ebc-f4eb-4242-9005-bd1a491fd4f4.PNG)

It works, but I observe, if set the Minimum SOC in EES setup greater than zero:

![ESS](https://user-images.githubusercontent.com/115069923/206924867-d4694fc6-10ad-45b0-bf71-d5debce09ccf.PNG)

- if enough SOC, everything is OK
- if set limit +1% reached, the discharge stops ... not exactly the set value, but OK
- next sunny morning a part or all of the Fronius power is fed into the grid instead of charging the batteries. Only the SmartSolar power is fully fed into the batteries
- if I set the SOC limit above the battery state, the whole (PVpower - consumption) flows into batteries

Do you have an idea whats is the reason? Does it concern the serial battery driver, or the ESS Algorithm?

Thanks a lot.

BR,
Anton

**Comment by Louisvdw** [comment]
Hi
The serialbattery driver does not look at the min SOC value in the system. It only sets the SOC value which is then used by the logic in the GX device.
So my though would be that this behaviour would be there if you were using any other battery as well. 

The best option might be the ask in the Victron forums how the charging algorithm works when you reach the min SOC with a Fronius in your system.
I am also curius to know if the battery is only charged from the SmartSolar for until some value above the min SOC is reached and then the Fronius kicks in, or if the Fronius is then never used for charging after that. This would indicate an algorithm is used or not.

**Reply by GeStenZ** [reply]
`It only sets the SOC value which is then used by the logic in the GX device.`

Interesting. How does it do this? - I was assuming so far that the SoC information is taken from the BMS and is not altered.

**Reply by Louisvdw** [reply]
Correct. The driver publish the SOC that is read from the BMS to the GX. The GX then has it own logic on how it use the SOC value to adjust other devices (MPPTs, inverters, etc.) in the system.

**Comment by mr-manuel** [comment]
Any new experiences after a few months?
